var/spack/repos/builtin/packages/piranha/package.py

```diff
@@ -0,0 +1,66 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Piranha(Package):
+    """Piranha is a computer-algebra library for the symbolic manipulation of
+    sparse multivariate polynomials and other closely-related symbolic objects
+    (such as Poisson series)."""
+
+    homepage = "https://bluescarni.github.io/piranha/sphinx/"
+    url      = "https://github.com/bluescarni/piranha/archive/v0.2.tar.gz"
+
+    # version('0.2', '4de7bd64f608cd66c1830956e3fa6f95')  # Not working ATM
+    version('develop', git='https://github.com/bluescarni/piranha.git')
+
+    variant('pyranha',   default=False,
```

Comment:
Look carefully - the variant name is different to the package name :-) This is what they call this library internally. I suppose I could rename it to `python` since it activates the python bindings. This would be consistent with what's done in the `deal.II` package, right?